cli/ballerina-cli/src/main/resources/new_cmd_defaults/tool-readme.md

```diff
@@ -0,0 +1,5 @@
+Introduces a custom command to the Ballerina CLI.
+[//]: # (above is the package summary)
```

Comment:
This line is not markdown compatible.

Reply:
Let's begin with the `Package overview`

Reply:
Fixed with c6dd5bf69b5a8d75058bab1374296152973ff3d1